ekncontent/ekncontent/eknc-domain.c

```diff
@@ -891,7 +938,7 @@ eknc_domain_query_finish (EkncDomain *self,
   QueryState *state = g_task_get_task_data (task);
   return g_object_new (EKNC_TYPE_QUERY_RESULTS,
                        "upper-bound", state->upper_bound,
-                       "models", state->models,
+                       "models", ptr_array_to_list_shallow (state->models),
```

Comment:
Going from a GPtrArray to a GList like this is not ideal, though I'm not sure if EkncQueryResults' properties are considered to be "external API" that we can't break.